robottelo/utils/virtwho.py

```diff
@@ -468,7 +468,9 @@ def hypervisor_json_create(hypervisors, guests):
                     "attributes": {"active": 1, "virtWhoType": "esx"},
                 }
             )
-        hypervisor = {"guestIds": guest_list}
+        hypervisor = {"hypervisorId": str(uuid.uuid4()), "guestIds": guest_list}
```

Comment:
I dont see `hypervisor` is not being reused below and we overriding hypervisor variable from line 473.

Can you please check that and fix it right ?

Reply:
Oh yeah, i left this line in. Lemme fix that. Good catch :)